lib/ClangImporter/ClangImporter.cpp

```diff
@@ -1246,7 +1245,7 @@ std::optional<std::vector<std::string>> ClangImporter::getClangCC1Arguments(
 
   std::vector<std::string> FilteredModuleMapFiles;
   for (auto ModuleMapFile : CI->getFrontendOpts().ModuleMapFiles) {
-    if (ctx.ClangImporterOpts.HasClangIncludeTreeRoot) {
+    if (ctx.CASOpts.HasImmutableFileSystem) {
```

Comment:
I'm not clear on this change -- isn't this still specific to the include tree?

Reply:
I guess so. I guess I didn't make the best distinction between this commit and the follow up I want to do, that is delete the CASFS clang modules (which only then they would be the same option).

Let me do it properly with a working intermediate version.